AlbumsNetCoreNoFramework/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Net;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var http = new HttpListener();
+
+var host = "127.0.0.1";
+var port = 5000;
+var contentType = "application/json";
+var concurrency = 10;
+
+http.Prefixes.Add($"http://{host}:{port}/");
+http.Start();
+
+var myAlbums = new[]
+{
+    new Album("1", "Pawn Hearts", "Van der Graaf Generator", 26.99),
+    new Album("2", "A Passion Play", "Jethro Tull", 17.99),
+    new Album("3", "Tales from Topographic Oceans", "Yes", 32.99)
+};
+
+#pragma warning disable 4014 // no await
+for(var i=0; i<concurrency; i++) http.GetContextAsync().ContinueWith(ProcessRequestHandler);
+#pragma warning disable 4014
+
+System.Console.ReadLine();
+
+async void ProcessRequestHandler(Task<HttpListenerContext> result)
+{
+    var context = await result;
```

Comment:
This needs to be disposed.

```suggestion
    using var context = await result;
```